frontend/src/components/common/input/styles.module.scss

```diff
@@ -24,6 +24,18 @@
   }
 }
 
+.textarea {
+  display: block;
+  padding: 11px 40px;
+  border: none;
+  border-radius: 7px;
+  outline: none;
+
+  &::placeholder {
+    color: rgb(40 40 40 / 50%);
+  }
+}
```

Comment:
```suggestion
.textarea {
resize: none;
}
```
and add it only for testarea

Reply:
should I delete all styles? It looks kinda raw
![image](https://user-images.githubusercontent.com/35200697/127992900-9e5d3d09-386f-4045-9467-98558694bfc5.png)